### PR TITLE
Fix: TreePicker Performance

### PR DESCRIPTION
**Pull request by camjc**
- New fastStatelessWrapper that wraps a stateless
  component so that it will only re-render on
  specified prop changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,3 +82,21 @@ Build Profiling
 To generate a profile: `npm run profile`
 
 Upload the new `stats.json` file to [Webpack Analyse Tool](http://webpack.github.io/analyse).
+
+Optimizing Performance of Stateless Components
+---------------
```
Comment (etaoins) [inline README.md]: :+1:

**Comment by etaoins** [comment]
LGTM!

**Comment by vinteo** [comment]
LGTM

**Comment by omgaz** [comment]
LGTM when you get it green

**Comment by vinteo** [comment]
Merging...

**Pull request merged**